infra/terraform/platform/main.tf

```diff
@@ -1,55 +1,44 @@
 terraform {
   backend "s3" {
     bucket = "terraform.analytics.justice.gov.uk"
-    key    = "dev/terraform.tfstate"
+    key    = "platform.tfstate"
     region = "eu-west-1"
   }
 }
 
 provider "aws" {
-  region = "${var.region}"
-}
-
-data "aws_caller_identity" "current" {}
-
-data "terraform_remote_state" "base" {
-  backend = "s3"
-
-  config {
-    bucket = "${var.terraform_bucket_name}"
-    region = "${var.region}"
-    key    = "${var.terraform_base_state_file}"
-  }
+  region  = "${var.region}"
+  version = "~> 1.19"
 }
 
 module "aws_vpc" {
-  source = "../../modules/aws_vpc"
+  source = "../modules/aws_vpc"
 
-  name               = "${var.env}.${data.terraform_remote_state.base.xyz_root_domain}"
+  name               = "${terraform.workspace}.${data.terraform_remote_state.base.xyz_root_domain}"
   cidr               = "${var.vpc_cidr}"
   availability_zones = "${var.availability_zones}"
 }
 
 module "cluster_dns" {
-  source = "../../modules/cluster_dns"
+  source = "../modules/cluster_dns"
 
-  env              = "${var.env}"
+  env              = "${terraform.workspace}"
   root_zone_name   = "${data.terraform_remote_state.base.xyz_dns_zone_name}"
   root_zone_domain = "${data.terraform_remote_state.base.xyz_root_domain}"
   root_zone_id     = "${data.terraform_remote_state.base.xyz_dns_zone_id}"
 }
 
 module "data_buckets" {
-  source = "../../modules/data_buckets"
+  source = "../modules/data_buckets"
 
-  env = "${var.env}"
+  env = "${terraform.workspace}"
 }
 
 module "user_nfs_softnas" {
-  source = "../../modules/user_nfs_softnas"
+  source = "../modules/user_nfs_softnas"
 
   num_instances             = 1
```

Comment:
Oh and I also noticed that we dropped the `default_volume_size = 250`, again, apologies if it's intended.

Reply:
Nope, that's all because `dev` and `alpha` config were slightly different, and this new version was taken from `dev` - but clearly I should've taken it from `alpha` - I'll fix.

Reply:
👍